src/xtgeo/grid3d/grid.py

```diff
@@ -1207,13 +1209,14 @@ def get_prop_by_name(self, name):
 
         return None
 
+    @deprecation.deprecated(
+        deprecated_in="2.7",
+        removed_in="3.0",
+        current_version=xtgeo.version,
+        details="Method get_cactnum is deprecated and can be removed",
```

Comment:
Do we have an optional method?

Reply:
I am not sure, the method has not returned any value since it was deprecated. Perhaps @jcrivenaes  can elaborate on which value it should have returned. Perhaps it should return something along the lines of `get_actnum`.